src/app/pages/card-import/card-import.page.ts

```diff
@@ -37,13 +42,18 @@ export class CardImportPage implements OnInit {
     targetHeight: 374
   }
 
-  constructor(private route: ActivatedRoute, private router: Router, private camera: Camera, private toastCtrl: ToastController, public alertController: AlertController, private cardService: CardService) {
+  constructor(private route: ActivatedRoute,
+              private router: Router,
+              private camera: Camera,
+              private toastCtrl: ToastController,
+              public alertController: AlertController,
+              private cardService: CardService) {
 
   }
 
    ngOnInit() {
        this.id = this.route.snapshot.paramMap.get('id');
-       console.log(this.id)
+       console.log(this.id);
```

Comment:
Copy changes from or rebase onto dev to get the changes here. When `this.id` exists, then the page should pull the corresponding card data. Currently, `this.card` does not exist.

Reply:
Yes, I saw you made that change in one of the branches previously merged branches. Admittedly, I didn't understand the purpose behind the change. I suggest a follow-up conversation where you specify these details so we all understand clearly.